### ISSUE TITLE
Update Language missing dialog to be more informative

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#416]
Inspired by [this forum post][1].  Showflash didn't realize they needed to configure Komodo to know where Python was or that Komodo looks on your system path for they Python interpreter.  See the Debug Error dialog in the 3rd image provided by Showflash.  That could easily convey the following to assist the user:

1. Mention we're looking on the path for the interpreter
2. Mention we look in the prefs and WHERE in the prefs (eg. Prefs > Languages > [lang]: Use this interpreter)
3. If we can, have the preferences button in the dialog link to the specific preference senction mentioned in point 2.

[1]:http://forum.komodoide.com/t/no-python-interpreter-could-be-found-error-round-ii/1786

**Comment by mitchell-as** [comment]
I really don't think this is necessary, because there's a "Preferences" button that immediately takes you to the Python language preferences pane with a very detailed description of the problem (see my screenshot).

![foo](https://cloud.githubusercontent.com/assets/12902397/14534872/0dd195f4-0239-11e6-9019-0a2c7d5b9519.png)

**Comment by Naatan** [comment]
I agree with @mitchell-as, but at the same time it would be very simple to add a line to the popup that says "Hit Preferences to configure an interpreter". The button is otherwise easy to miss.
